File: examples/TemplateEditExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

$template_edit_request = (new Dropbox\Sign\Model\TemplateEditRequest())
    ->setCcRoles([
        "Role 1",
        "Role 2",
    ]);

try {
    $response = (new Dropbox\Sign\Api\TemplateApi(config: $config))->templateEdit(
        template_id: "f57db65d3f933b5316d398057a36176831451a35",
        template_edit_request: $template_edit_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling TemplateApi#templateEdit: {$e->getMessage()}";
}

File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateEditExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateEditExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var templateEditRequest = new TemplateEditRequest(
            ccRoles: [
                "Role 1",
                "Role 2",
            ]
        );

        try
        {
            var response = new TemplateApi(config).TemplateEdit(
                templateId: "f57db65d3f933b5316d398057a36176831451a35",
                templateEditRequest: templateEditRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling TemplateApi#TemplateEdit: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
